src/startupMessage.js

```diff
@@ -7,6 +7,11 @@ export const nextstrainAbout = `
 `;
 
 export async function startupMessage(server, app, groupsApp) {
+
+  const [awsId] = (await Promise.allSettled([
+    groupsApp ? _false() : getAwsId(), // omit AWS user for groups app
```

Comment:
AWS credentials will still be configured for the Groups app, so I'm not sure why we're skipping this? The credentials may not be true AWS credentials, but credentials for an S3-compatible service (like Cloudian Hyperstore, Minio, etc). Some of those will also support an STS-compatible service for GetCallerIdentity, but some won't.

Reply:
> AWS credentials will still be configured for the Groups app, so I'm not sure why we're skipping this? 

Skipped due to (PR feedback)[https://github.com/nextstrain/nextstrain.org/pull/826#discussion_r1562968002]

Reply:
I didn't realize that it would be useful beyond AWS. If that's the case, I'm fine with it being included as long as the wording doesn't assume AWS

Reply:
Nod. I think we'd do well to be precise with wording here. CDC, for example, runs the Groups app against a Cloudian Hyperstore's S3-compatible service. But they still set `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` (and some other things) in order to authenticate with it, because we're still using an AWS library to talk to it.

Reply:
Repushed to go back to the original implementation (i.e. we always attempt to get the AWS(-like) ID)